homeassistant/components/template/strings.json

```
@@ -2,6 +2,7 @@
   "common": {
     "advanced_options": "Advanced options",
     "availability": "Availability template",
+    "availability_description": "Defines a template to get the `available` state of the entity. If the template either fails to render or returns `True`, `\"1\"`, `\"true\"`, `\"yes\"`, `\"on\"`, `\"enable\"`, or a non-zero number, the entity will be `available`. If the template returns any other value, the entity will be `unavailable`. If not configured, the entity will always be `available`. Note that the string comparison is not case sensitive; `\"TrUe\"` and `\"yEs\"` are allowed.",
     "code_format": "Code format",
     "device_class": "Device class",
```

You defined common strings for those, so let's use 'em here now, too:

```suggestion
    "code_format": "[%key:component::template::common::code_format%]",
    "device_class": "[%key:component::template::common::device_class%]",
```

---

But that's the common section